### PR TITLE
testing/s-nail: new aport

### PR DESCRIPTION
**Pull request by pickfire**

### CODE REVIEW

**Comment by pickfire** [comment]
Oh no, I have accidentally overwritten s-nail. T_T

**Comment by pickfire** [comment]
Luckily I somehow managed to recover it by checking `git reflog`.

**Comment by pickfire** [comment]
Luckily I somehow managed to recover it by checking `git reflog`.

File: testing/tlp/tlp.initd [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+# TODO: I don't know how to write this initd
+case "$1" in
+    start|\
+    stop|\
+    restart|\
+    force-reload)
+        tlp init $1
+        ;;
+esac
```
Comment (jirutka) [inline testing/tlp/tlp.initd]: This is totally wrong. OpenRC is not SysV init.

Comment (pickfire) [inline testing/tlp/tlp.initd]: https://github.com/alpinelinux/aports/commit/c860144b5b79ea667fbedcc34a800637b0339b88#diff-e74cce27af5d72760a36be525d08364eR17

**Comment by pickfire** [comment]
How? It is already in master. GG

**Comment by pickfire** [comment]
Horde of patches are coming!!!

**Comment by jirutka** [comment]
IIRC I've asked you to separate at least Firefox and other big packages into a separate PR... In this way it's useless, because will not build many pkgs.

**Comment by jirutka** [comment]
If you don’t have https://github.com/alpinelinux/aports.git in remotes yet:

```sh
git remote add upstream https://github.com/alpinelinux/aports.git
```

Create a new branch based on the current upstream/master:it.

```sh
git fetch upstream
git checkout upstream/master
git checkout -b update-py
```

Cherry-pick your current py-* commits into the new branch (021b52d… is SHA of your first commit in this PR, 16948a8… is SHA of your last commit in this PR):

```sh
git log --pretty=oneline 021b52d897114a324f3977f3e23490fafc0a64ad..16948a858c08b348a45a00916530d20ce68add54 | sed -En 's|^(\w+) \w+/py-.*|\1|p' | xargs -n 1 git cherry-pick
```

Push to your “fork” on GitHub:

```sh
git push -u origin update-py
```

Open a PR from your branch update-py to master in this repository. You can do it even from CLI, e.g. using [Idnan/github-pr](https://github.com/Idnan/github-pr) (but don’t install it via curl | sudo!) or similar tool.

**Comment by pickfire** [comment]
I have created a few branches just for checking. This can be closed.

**Pull request closed without merging**